### PR TITLE
TS-4604 : Command for configuring traffic server corrected

### PR DESCRIPTION
**Pull request by persiaAziz**

### CODE REVIEW

File: doc/admin-guide/configuring-traffic-server.en.rst [hunk 1]
```diff
@@ -27,13 +27,13 @@ Traffic Server provides several options for configuring the system.
 
 .. _configure-using-traffic-line:
 
-Configure Traffic Server Using Traffic Line
+Configure Traffic Server
 ===========================================
 
 Traffic Line enables you to quickly and easily change your Traffic
```
Comment (jpeach) [inline doc/admin-guide/configuring-traffic-server.en.rst]: Correct this line too?

File: doc/admin-guide/configuring-traffic-server.en.rst [hunk 2]
```diff
@@ -43,13 +43,13 @@ To view a configuration setting, enter the following command::
 where *var* is the variable associated with the configuration
 option. For a list of variables, refer to :ref:`configuration-variables`.
 
-Change Configuration Options in Traffic Line
+Change Configuration Options
```
Comment (jpeach) [inline doc/admin-guide/configuring-traffic-server.en.rst]: I think the underlines need to be the same length as the headline text.

Comment (persiaAziz) [inline doc/admin-guide/configuring-traffic-server.en.rst]: Sure

File: doc/admin-guide/configuring-traffic-server.en.rst (hunk 1 above)
Comment (persiaAziz) [inline doc/admin-guide/configuring-traffic-server.en.rst]: Oh yes, I missed that

File: doc/admin-guide/configuring-traffic-server.en.rst [hunk 3]
```diff
@@ -27,14 +27,14 @@ Traffic Server provides several options for configuring the system.
...
 
-Configure Traffic Server Using Traffic Line
-===========================================
+Configure Traffic Server
+========================
 
-Traffic Line enables you to quickly and easily change your Traffic
-Server configuration via command-line interface.
+You can quickly and easily change your Traffic
+Server configuration via command-line interface using 'traffic_ctl'.
```
Comment (jpeach) [inline doc/admin-guide/configuring-traffic-server.en.rst]:
```
program:`traffic_ctl`
```

This syntax auto-links to the traffic_ctl reference.

**Pull request merged**